### PR TITLE
[DEV-1634] submodule poc

### PR DESCRIPTION
**Pull request by tommaso1**
DEV-1514

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "download-docs": "./scripts/fetch-docs.sh docs/from-gitbook",
+    "update-docs": "git submodule init && git submodule update && cd docs && git checkout docs/from-gitbook && git pull",
```
Comment (marcobottaro) [inline apps/nextjs-website/package.json:7]: You should move this commands into a shell script and use the temporary folder to `move` only `docs` subfolder in `apps/nextjs-website/docs/`. Right now with this script you create the nested subfolder `docs/docs` and the parser cannot access the manuals correctly.

File: .gitignore [hunk 2]
```diff
@@ -392,6 +392,5 @@ docker/
 
 # Ignore files managed by gitbook fetch step
 apps/nextjs-website/.tmp/
```
Comment (MarcoPonchia) [inline .gitignore:394]: I think docs folder shouldn't be pushed to repository and is correct to be ignored or there is something i have missed ? :thinking: 
```suggestion
apps/nextjs-website/.tmp/
apps/nextjs-website/docs/
```

**Review by MarcoPonchia — CHANGES_REQUESTED** [review]

**Review by marcobottaro — CHANGES_REQUESTED** [review]
You should add the changeset file

File: apps/nextjs-website/scripts/fetch-local-docs.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+git submodule init
+git submodule update
+cd  docs 
+# discard all untracked changes
+git clean -fd
+git checkout docs/from-gitbook 
+git pull
+# copy all files to root ovveriding existing files
+cp -r docs/* .
```
Comment (marcobottaro) [inline apps/nextjs-website/scripts/fetch-local-docs.sh:9]:
```suggestion
mv docs/* .
rm -R docs/

```

**Review by marcobottaro — CHANGES_REQUESTED** [review]

**Review by marcobottaro — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -19,6 +19,15 @@ npm i
 npm run download-docs -w nextjs-website
 ```
 
+#### How to speed up local docs fetch time
+
+Initialize a git submoudule on docs directory:  
+`rm -R apps/nextjs-website/docs && git submodule add --force https://github.com/pagopa/devportal-docs.git apps/nextjs-website/docs`  
+
+Run the script to checkout docs to the right branch
+`npm run update-docs -w nextjs-website`
```
Comment (marcobottaro) [inline README.md:28]: You didn't add the two options. If a developer runs the commands in the order written in the README file, they will download a gigabyte of data and then delete it

**Review by marcobottaro — CHANGES_REQUESTED** [review]

**Review by marcobottaro — APPROVED** [review]

**Review by MarcoPonchia — APPROVED** [review]

**Review by christian-calabrese — APPROVED** [review]
LGTM

**Pull request merged**